test/common/upstream/load_balancer_fuzz.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package test.common.upstream;
+
+import "validate/validate.proto";
+import "envoy/config/cluster/v3/cluster.proto";
+import "google/protobuf/empty.proto";
+
+message UpdateHealthFlags {
+  uint64 host_index = 1;
```

Comment:
Please add comments to explain what these fields do.

Reply:
Added.